src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/TextAreaPart.cshtml

```diff
@@ -18,4 +18,4 @@
         }
     }
 }
-<textarea id="@fieldId" name="@fieldName" class="@fieldClass" placeholder="@Model.Value.Placeholder">@fieldValue</textarea>
+<textarea id="@fieldId" name="@fieldName" class="@fieldClass" placeholder="@Model.Value.Placeholder" rows="@Model.Value.Rows">@fieldValue</textarea>
```

Comment:
Only add rows= attribute if Model.Value.Rows.HasValue

Reply:
We could use rows="3" if the `Rows` is `null`

Reply:
We should not force any number since that is an opinionated value. I would have no value by default to allow one to style the default using CSS. However, adding an option for someone to provide # of rows to provide a value is a good idea